cmp/grid/Grid.js

```diff
@@ -154,7 +162,7 @@ export class Grid extends Component {
             frameworkComponents: {agColumnHeader: ColumnHeader},
             rowSelection: model.selModel.mode,
             rowDeselection: true,
-            getRowHeight: () => model.compact ? Grid.COMPACT_ROW_HEIGHT : Grid.ROW_HEIGHT,
+            getRowHeight: () => Math.max(this.columnRowHeight, model.compact ? Grid.COMPACT_ROW_HEIGHT : Grid.ROW_HEIGHT),
```

Comment:
would work this extra bit of max logic in to the computed header -- just cleaner to do all the rowheight logic in one place